server/app/assets/javascripts/preview.ts

```diff
@@ -463,11 +463,23 @@ class PreviewController {
 
     items.forEach((item) => {
       const listItem = document.createElement('li')
-      listItem.textContent = item
+      listItem.appendChild(PreviewController.parseMarkdown(item))
       listTag.appendChild(listItem)
     })
     return listTag
   }
+
+  private static parseMarkdown(currentLine: string): Element {
+    const md = new MarkdownIt()
+    const parser = new DOMParser()
```

Comment:
Do you know what the recommended practice for instantiating these two object are? I see `parseMarkdown` used in a tight loop and I just want to be cognizant of memory and garbage collection. Does the static typescript method get transpiled to not make it an issue?

Reply:
Great question, and I don't know. I feel like instantiating an object in a tight loop is probably not the recommended practice though so I'll do some digging. That's for catching/calling this out! The examples in the docs instantiate it right below the import statement which felt weird to me but might be the way to do it.